rubrik_cdm/cluster.py

```diff
@@ -20,9 +20,76 @@
 
 
 class Cluster(Api):
-    """This class contains methods related to the managment of the Rubrik cluster itself.
+    """This class contains methods related to the management of the Rubrik cluster itself.
     """
 
+    def configure_replication_private(self, username, password, target_ip,
+                              ca_certificate=None, timeout=30):
+        """Configure replication partner as specified by user using PRIVATE NETWORK (direct connection)
+        Keyword Arguments:
+            -MANDATORY:
+            username: Username for the TARGET cluster {string}
+            password: Password for the TARGET cluster {string}
+            target_ip: Address of one of the nodes of the TARGET cluster {string}
+            -OPTIONAL:
+            CaCert: optional CA Certificate
+            timeout: default 30, timeout for eventual API call
+        """
+
+        config = {}
+
+        config['replicationSetup'] = 'Private Network'
+        config['targetClusterAddress'] = target_ip
+        config['username'] = username
+        config['password'] = password
+
+        if ca_certificate is not None:
+            config["caCerts"] = ca_certificate
+
+        self.log("configure_replication: Adding cluster '{}' as private network replication target.".format(target_ip))
+
+        return self.post("internal", "/replication/target", config, timeout)
+
+    def configure_replication_nat(self, username, password, src_gateway, tgt_gateway,
```

Comment:
`src_gateway` -- `tgt_gateway`

We can be more verbose here. `source_gateway` and `target_gateway` are going to be even more obvious